devel/Dockerfile

```diff
@@ -35,6 +38,21 @@ RUN git clone --recursive https://github.com/phoenix-rtos/phoenix-rtos-project.g
     cd .. && rm -rf phoenix-rtos-project && \
     chmod -R a+rwX /opt/phoenix/utils
 
+
+#install qemu from xilinx
+RUN git clone https://github.com/Xilinx/qemu.git && \
+    cd qemu && \
+    git checkout tags/xilinx_v2021.2 && \
+    git submodule update --init dtc && \
+    mkdir build && cd build && \
+    ../configure --target-list="aarch64-softmmu" --enable-fdt --disable-kvm --disable-xen --enable-gcrypt && \
+    make -j4 && \
```

Comment:
we internally hardcode -j9 in `phoenix-rtos-project` so there is no need to be prude now :).

Either go with `-j9` or `-j$(($(nproc)+1))` if You want to be fancy :clinking_glasses:

Reply:
Could you provide some source for why to use 
`-j$(($(nproc)+1))` and not `-j$(($(nproc)))`?
I would love to know the arguments for this `+1`.
For processors that have 4, 6 or 8 threads `-j 9` is not a good solution.

Reply:
The origin of this convention is the (generally) correct assumption that compiling process is mostly I/O bound and not CPU bound. Most of the compilation is reading multiple files (`.c` + many `.h` per compilation unit) + writing output (mostly `.o` files). As the compilation process enters blocking I/O operation, it frees the CPU core and it can start executing some other READY process. As I/O devices were slow, it was advised to have `NPROC` ready-to-run processes and one I/O-bound at any particular moment to achieve best compilation time (note: not best system responsiveness during compilation :)).

Currently (because of SSDs, compilation from ramdisks, ccache,  and some of the optimizations becoming CPU-heavy), this is mostly anecdotal and greatly varies from project to project (eg. `phoenix-rtos-project` -without `ports` - has roughly the same compilation time with `-j8` `-j9`, `-j12` (7.8-8.0s), but seems slower for `-j4` and `-j16` (8.6-8.8s) on my machine).

I believe `-j9` to be sane default for most modern hardware, some special cases (eg. building in CI on lightweight VMs) might benefit from some special options. Please also note that GNU make has `-l` option to limit job spawning if the load exceeds certain amount (so if there are too many CPU-bound tasks right now), but I haven't really experimented with that ;).

Reply:
Fixed